src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -1566,6 +1666,143 @@ private SyntaxNode WithModifiersInternal(SyntaxNode declaration, DeclarationModi
             }
         }
 
+        public override SyntaxNode AsAbstractMember(SyntaxNode declaration)
+        {
+            return Isolate(declaration, AsAbstractMemberInternal);
+        }
+
+        private SyntaxNode AsAbstractMemberInternal(SyntaxNode declaration)
+        {
+            var newMods = GetModifiers(declaration).WithIsVirtual(false).WithIsOverride(false).WithIsAbstract(true).WithIsNew(false).WithIsSealed(false);
```

Comment:
I see what you're doing below.  I so so with how this works.  Just consider if we want to treat 'new' separately from the other modifiers.